tests/jinja2/Jinja2BearTest.py

```diff
@@ -164,6 +164,26 @@ def test_control_spacing(self):
     valid_files=(good_file1, good_file2, good_file3),
     invalid_files=(bad_file1, bad_file2))
 
+good_file_01 = """
```

Comment:
change the variable name to "valid_file_without_end_comments" or something like that.